packages/frontend-utils/src/events/index.ts

```diff
@@ -52,14 +50,6 @@ export const getEventListOptions = <C extends Constraint = EventConstraint>(
     constraint,
     pageSize,
     filters: new Set(),
-    ...(past
-      ? {
-          before: time,
-          sort: {
-            sortBy: 'endDate',
-            sortOrder: 'desc' as const,
-          },
-        }
-      : { after: time }),
+    ...(past ? { before: time } : { after: time }),
```

Comment:
how does the ordering got changed?

Reply:
1. The sort is no longer used by crn, the events are sourced from Algolia and this property is no longer used.
2. The past events should be ordered by endData desc. GP2 wasn't doing this.

Reply:
and we are gonna do it now?

Reply:
I'm just trying to understand if there is work missing for events